src/config/prism.json

```diff
@@ -1,5 +1,9 @@
 {
   "country": "Mongolia",
+  "serversUrls": {
+    "wcs": ["http://185.30.8.6:5000"],
```

Comment:
So technically in Mongolia we have 2 servers with base URLS:

geoserver: https://mng-wfp.ovio.org:8443/geoserver/prism/
openDataCube: http://185.30.8.6:5000/


They can **both** return WMS & WCS layers using /wms?... or /wcs?... . The difference is that they don't return GetCapabilities the same way for WMS, which is why the python function I shared with you was a bit convoluted.

Reply:
We will be using `geoserver` only for WMS though and `openDataCube` for both.
I think we should restructure the layers in `layers.json` to take:

- `server_id` instead of the `server_uri`
- `server_type`
- `server_layer`
- `server_style` (optional)

So that we can build URLs like this one programmatically -> `http://185.30.8.6:5000/wms?styles=ndvi&layers=ModisIndices`

Reply:
Thoughts @bencpeters ?